av1-id3.md

```diff
@@ -105,6 +105,8 @@ aligned(8) class DASHEventMessageBox extends FullBox('emsg', version, flags = 0)
 
 `message_data` contains complete ID3 version 2.4 data \[[ID3]\].
 
+The id field is not restricted in this version of the specification.

```

Comment:
minor editorial point. You can use quotes to ensure that the id field is styled as a syntactical element.